### ISSUE TITLE
Opening an ogg file results in double-free

### ISSUE THREAD

**Issue by pmiddend** [Zuzu-Typ/PyOgg#10]
I'm trying to open an OGG file not for streaming, but "normally":

```
VorbisFile("somefile.ogg")
```

This fails with:

```
double free or corruption (out)
```

Using gdb, I found out that it's somewhere below `ov_fopen` inside the vorbis library.

Are the vorbis bindings outdated maybe? Or just wrong? I tested this using multiple distributions, using the latest version of PyOgg and version `1.3.6` ob `libvorbis`.

The gdb trace is:

```
#0  0x00007ffff649fbe0 in raise ()
   from /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/libc.so.6
#1  0x00007ffff64a0dc1 in abort ()
   from /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/libc.so.6
#2  0x00007ffff64e12ac in __libc_message ()
   from /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/libc.so.6
#3  0x00007ffff64e749a in malloc_printerr ()
   from /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/libc.so.6
#4  0x00007ffff64e83e0 in _int_free ()
   from /nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27/lib/libc.so.6
#5  0x00007fffeb173c37 in vorbis_staticbook_destroy ()
   from /nix/store/a89n6j27jd97wzywkzxfq36zy222krfc-libvorbis-1.3.6/lib/libvorbis.so
#6  0x00007fffeb15da05 in vorbis_synthesis_init ()
   from /nix/store/a89n6j27jd97wzywkzxfq36zy222krfc-libvorbis-1.3.6/lib/libvorbis.so
#7  0x00007fffead1e140 in _fetch_and_process_packet.constprop.14 ()
   from /nix/store/a89n6j27jd97wzywkzxfq36zy222krfc-libvorbis-1.3.6/lib/libvorbisfile.so
#8  0x00007fffead22c28 in ov_read_filter ()
   from /nix/store/a89n6j27jd97wzywkzxfq36zy222krfc-libvorbis-1.3.6/lib/libvorbisfile.so
#9  0x00007fffead23011 in ov_read ()
```

**Comment by Zuzu-Typ** [comment]
Does this issue still persist?

**Comment by pmiddend** [comment]
Sure, but we switched to opus, so it doesn't bother us anymore.

**Comment by Zuzu-Typ** [comment]
Might have been fixed by de31f2cf73418d68575d82a391e2e97507803a82 

Please reopen if the issue still persists.
